skimage/segmentation/tests/test_active_contour_model.py

```diff
@@ -112,35 +104,27 @@ def test_bad_input():
     c = np.linspace(5, 424, 100)
     init = np.array([r, c]).T
     with testing.raises(ValueError):
-        active_contour(img, init, boundary_condition='wrong',
-                       coordinates='rc')
+        active_contour(img, init, boundary_condition='wrong')
     with testing.raises(ValueError):
-        active_contour(img, init, max_iterations=-15,
-                       coordinates='rc')
+        active_contour(img, init, max_iterations=-15)
 
 
-def test_bc_deprecation():
-    with expected_warnings(['boundary_condition']):
-        img = rgb2gray(data.astronaut())
-        s = np.linspace(0, 2*np.pi, 400)
-        r = 100 + 100*np.sin(s)
-        c = 220 + 100*np.cos(s)
-        init = np.array([r, c]).T
+def test_xy_coord_warning():
```

Comment:
I would use `for` loop to test for `xy` and `None` and rename the test:
```suggestion
def test_coord_raises():
```